client/src/pages/spectrogram/SpectrogramPage.tsx

```diff
@@ -305,15 +304,7 @@ export const SpectrogramPage = () => {
   };
 
   return (
-    <SpectrogramContext.Provider
```

Comment:
Just a curiosity - how come we're not using the context component anymore?

Reply:
It was never used by any component